src/components/Carousel/Carousel.js

```diff
@@ -85,6 +67,8 @@ function Carousel(props) {
   let currentSlideData = questionsData?.filter(
     (item) => item.order === currentSlide
   )
+  let questionConfig =
+    currentSlideData && currentSlideData[0]?.questionConfigurations
```

Comment:
for the future, you can replace that expression on currentSlideData?.[0]?.questionConfigurations result will be the same